test/e2e/framework/skip.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// This file will be removed after switching to use subpackage skip.
```

Comment:
Please make this file import the new subpackage and re-use it. E.g. `func Skipf(format string, args ...interface{}) {` should become `var Skipf = e2eskip.Skipf`.

That way there's no risk of divergent changes.

Reply:
...in fact, you could do that in a separate change before modifying the e2e tests at all, if you like.

Reply:
@lavalamp Thanks for your review, that is a nice idea.
I will update the PR based on your point.

Reply:
@lavalamp Oops, that makes circular dependency between framework package(old one) and skip package(new one) because a new skip package needs to import framework package.

Reply:
In that case, make the e2eskip package contain the redirections?

Reply:
Technically we can do that, but I feel it is easier to just remove the framework package's skip.go file after all callers switch to use e2eskip package instead of we need to change e2eskip package.

Reply:
It's not really OK to have all this code copied in the mean time, though.

Reply:
OK, I will update this PR to make e2eskip package redirect to framework package.

Reply:
@lavalamp
Thanks for your review.
I think your comments are applied to the latest PR.
Could you check this again?